building.md

```diff
@@ -0,0 +1,86 @@
+# Build Vita3K
+
+## Windows (Visual Studio)
+
+- Install Visual Studio 2017 or 2019 and choose to install `Desktop development with C++` (as shown in this [image](https://i.imgur.com/bkY15Oh.png)). You will get compiler and `cmake` required for building.
```

Comment:
Maybe we should have an `images` folder. We are already using other images in README.

Reply:
I think we shouldn't include binary images in the repo as it might be changed and increase the repo size significantly. Yeah, someone might remove the images or the hosting providers will remove our images sometime in the future. But these images are not essential to our project so we can replace them anytime we want.